sass/components/_inputs.scss

```diff
@@ -1,9 +1,7 @@
 @charset 'UTF-8';
 
 .help-text {
```

Comment:
Can it mess the things up in the admin console? the ".help-text" is used in other places a lot.

Reply:
True, I can add it back again with a minor change.